element_animal/surgery.py

```diff
@@ -0,0 +1,169 @@
+    """
+    contents = zip(
+        ["bregma", "lambda", "dura", "skull_surface", "sagittal_suture", "sinus"]
+    )
+
+
+@schema
+class BrainRegion(dj.Manual):
+    """Brain region of a given surgery
+
+    Attributes:
+        region_acronym ( varchar(32) ) : Brain region shorthand
+        region_name ( varchar(128) ) : Brain region full name
+    """
+
+    definition = """
+    region_acronym : varchar(32)   # Brain region shorthand
+    ---
+    region_name    : varchar(128)  # Brain region full name
+    """
+
+
+@schema
+class Hemisphere(dj.Lookup):
+    """Brain region hemisphere
+
+    Attributes:
+        hemisphere ( varchar(8) ): Brain region hemisphere (e.g., left, right, middle)
+    """
+
+    definition = """
+    hemisphere: varchar(8) # Brain region hemisphere
+    """
+
+    contents = zip(["left", "right", "middle"])
+
+
+@schema
+class ImplantationType(dj.Lookup):
+    """Type of implantation
+
+    Attributes:
+        implant_type ( varchar(16) ): Short name for type of implanted device
+        implant_description ( varchar(32) ): Full description for implanted device
+    """
+
+    definition = """
+    implant_type        : varchar(16) # Short name for type of implanted device
+    ---
+    implant_description : varchar(32) # Full description for implanted device
+    """
+
+    contents = (
+        ("ephys", "electophysiology"),
+        ("fiber", "fiber photometry"),
+        ("opto", "optogenetic pertubation"),
+    )
+
+
+@schema
+class Implantation(dj.Manual):
+    """Implantation of a device
+
+    WRT: With Respect To
+
+    Attributes:
+        Session (foreign key): Session primary key
+        location_id (int): ID of of brain location
+        ap ( decimal(6, 3) ): In um, Anterior/posterior; Anterior Positive
+        ap_reference (projected attribute): Coordinate reference
+        ml ( decimal(6, 3) ): In um, medial axis; Right Positive
+        ml_reference (projected attribute): Coordinate reference
+        dv ( decimal(6, 3) ): In um, dorso-ventral axis. Ventral negative
+        dv_reference (projected attribute): Coordinate reference
+        theta ( decimal(6, 3), nullable ): Elevation in degrees.
+            Rotation about ml-axis [0, 180] WRT Z
+        phi ( decimal(6, 3), nullable ): Azimuth in degrees.
+            Rotations about dv-axis [0, 360] WRT X
+        beta ( decimal(6, 3), nullable ): Rotation about shank in degrees.
+            Rotation about the shank [-180, 180]. Clockwise is increasing.
+            0 is the probe-front facing anterior
+    """
+
+    definition = """
+    -> subject.Subject
+    implant_date  : datetime       # surgery date
+    -> ImplantationType
+    -> BrainRegion                 # targeted brain region for this implantation
+    -> Hemisphere                  # targeted hemisphere for this implantation
+    ---
+    -> User.proj(surgeon='user')   # surgeon
+    ap            : decimal(6, 3)  # (um) anterior-posterior; ref is 0
```

Comment:
```suggestion
    ap            : decimal(6, 3)  # (μm) anterior-posterior; ref is 0
```

can we use μm?

Reply:
I don't know that it would cause issues, but i'm reticent to include nonstandard characters in table definitions, even comments. I'd hate for a change in mysql backend to break table design because of an esoteric character encoding issue